src/main/kotlin/net/maizegenetics/phgv2/api/ExportHaplotypeGraph.kt

```diff
@@ -47,16 +53,44 @@ fun exportMultiSampleHVCF(
         .build()
         .use { writer ->
 
+            val rangeStrToIndex = graph.refRangeStrToIndexMap()
+            val checksumToId = mutableMapOf<String, String>()
+
             val headerLines = graph.altHeaders().values
-                .map { altHeaderMetadataToVCFHeaderLine(it) }
+                .map {
+                    when (symbolicAllele) {
+                        SymbolicAllele.CHECKSUM -> {
+                            checksumToId.putIfAbsent(it.checksum, it.id)
+                            altHeaderMetadataToVCFHeaderLine(it)
+                        }
+
+                        SymbolicAllele.RANGE_SAMPLE_GAMETE -> {
+                            altHeaderMetadataToVCFHeaderLine(
+                                it,
+                                symbolicAlleleRangeSampleGameteStr(
```

Comment:
Will you clarify the number given for the reference range?  If the graph was made with only specific ranges, the range numbers will be different than if a graph was made with all the reference ranges?   Is data comparable between multiple exported VCFs, or does it depend on how the graphs were made?